arthur/tasks.py

```diff
@@ -124,13 +125,20 @@ def has_resuming(self):
 
         return self._has_resuming
 
+    def set_job(self, job_id, job_number):
```

Comment:
Can you add some documentation for this method?

Reply:
Done :+1: